src/functionHandlers/notarisePdt/v2/notarisePdt.ts

```diff
@@ -23,14 +22,13 @@ export const notarisePdt = async (
   certificate: WrappedDocument<PDTHealthCertV2>,
   parsedFhirBundle: ParsedBundle,
   testData: TestData[]
-): Promise<{ result: NotarisationResult; directUrl: string }> => {
+): Promise<NotarisationResult> => {
   const errorWithRef = trace.extend(`reference:${reference}`);
   const traceWithRef = trace.extend(`reference:${reference}`);
 
   const { id, key } = await getQueueNumber(reference);
   traceWithRef(`placeholder document id: ${id}`);
 
-  const directUrl = buildStoredDirectUrl(id, key);
   const storedUrl = buildStoredUrl(id, key);
```

Comment:
Since `buildStoredUrl()` calls `universalUrl()`, shall we rename it to `buildUniversalUrl()`?

Reply:
`storedUrl` can also be renamed accordingly